Source/Clients/DotNET/Webhooks/AuthenticationType.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Cratis. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cratis.Chronicle.Webhooks;
+
+/// <summary>
+/// Represents the type of authentication supported by a webhook.
+/// </summary>
+public enum AuthenticationType
```

Comment:
Suggestion; rename it to AuthorizationType - since its not about identity, but access.